octoprint_octorelay/__init__.py

```diff
@@ -126,19 +126,20 @@ def handle_get_status_command(self, index: str):
         self._logger.debug(f"Responding to {GET_STATUS_COMMAND} command: {is_closed}")
         return flask.jsonify(status=is_closed)
 
-    def handle_update_command(self, index: str):
-        self._logger.debug(f"Requested to switch the relay {index}")
+    def handle_update_command(self, index: str, target: Optional[bool] = None):
+        self._logger.debug(f"Requested to switch the relay {index} to {target}")
         if not self.has_switch_permission():
             self._logger.warn("Insufficient permissions")
             return flask.abort(403)
         if index not in RELAY_INDEXES:
             self._logger.warn(f"Invalid relay index supplied: {index}")
             return flask.jsonify(status="error")
-        self.toggle_relay(index)
+        target_bool = True if target == "ON" else False if target == "OFF" else None
+        relay_result = self.toggle_relay(index, target_bool)
```

Comment:
excessive statement, since `target` is `Optional[bool]` (either `bool` or `None`)

```suggestion
        relay_result = self.toggle_relay(index, target)
```

Reply:
removed in 900519a